client/state/test/initial-state.js

```diff
@@ -10,17 +11,26 @@ import sinon from 'sinon';
 import config from 'config';
 
 describe( 'initial-state', () => {
-	let localforage, createReduxStoreFromPersistedInitialState, MAX_AGE;
+	let localforage, createReduxStoreFromPersistedInitialState, MAX_AGE,
+		isSwitchedUser = false,
+		isSupportUserSession = () => isSwitchedUser;
 
 	require( 'lib/react-test-env-setup' ).auto();
 
 	before( () => {
+		mockery.registerMock( 'lib/user/support-user-interop', { isSupportUserSession: isSupportUserSession } );
+		mockery.enable( { warnOnUnregistered: false } );
```

Comment:
This needs to spec `useCleanCache: true` since you can no longer guarentee that what you're mocking hasn't been required by someone else in a previous test run with the single runner. Or just use `useMockery` from `test/use-mockery` and it'll be taken care of for you.